### ISSUE TITLE
Snapshot importing user failure

### ISSUE THREAD

**Issue by jmilum** [wesabe/pfc#2]
after applying commit 361d28a288d3ebfce363073f946ac8a1c4b9a046, I am now getting this error when I try to import my snapshot:

```
unzipping snapshot...
importing user...
/Users/jeremy/projects/wesabe/pfc/app/models/user/current_user_concern.rb:9:in `current=': Invalid user. Expected an object of class 'User', got true (ArgumentError)
        from /Users/jeremy/projects/wesabe/pfc/lib/importer/wesabe.rb:34:in `import'
        from /Users/jeremy/projects/wesabe/pfc/lib/importer/wesabe.rb:12:in `import'
        from /Users/jeremy/projects/wesabe/pfc/lib/tasks/snapshot.thor:10:in `import'
        from /Library/Ruby/Gems/1.8/gems/thor-0.13.8/lib/thor/task.rb:22:in `send'
```

I ran the import successfully before that commit, and it failed after.

**Comment by eventualbuddha** [comment]
Verified this, working on a fix.

**Comment by eventualbuddha** [comment]
No need to change the password. Closed by 6671838ab8715905e9053301b25af0ce52f07bbf.
